scripts/generateContent.gradle

```diff
@@ -0,0 +1,74 @@
+buildscript {
+    repositories {
+        maven {
+            credentials {
+                username mavenUsername
+                password mavenPassword
+            }
+            url mavenRepository
+        }
+    }
+    dependencies {
+        classpath 'com.theokanning.openai-gpt3-java:service:0.12.0'
+    }
+}
+
+import com.theokanning.openai.service.OpenAiService;
+import com.theokanning.openai.completion.CompletionRequest;
+
+//tag::generateContent[]
+task generateContent (
+    description: 'Generate documentation with the help of OpenAI',
+    group: 'docToolchain'
+) {
+    doLast{
+        def token  =  (findProperty("openAI.token")?:config.openAI.token)?:null
+        if(token == null){
+            throw new Exception ("""
+            >> No Token provided for openAI.
+            >> Please specify the token using -PopenAI.token=xx-xxxxxxxxxxxxxxxx or in your docToolchainConfig.groovy
+            """)
+        }
+        def model = config.openAI.model?:"text-davinci-003"
+        def maxToken = config.openAI.maxToken?:500
+        def temperature = config.openAI.temperature?:0.3
```

Comment:
Would it not make sense to provide means to overrule all configure parameters via command line?
Trying out stuff is less painful.

I know that not all tasks provide this feature. But we want to go into this direction. So let's make new tasks as we would like to have them

Reply:
I initially thought about seperating config values that are required to even execute the task, like the token, and config params that ay be more advanced settings like these. 
I'm fine if we default to parameter overwrite always possible. This way we could have a consistent configuration mechanism that follows precedence like:
1. via CLI (always win)
2.  via config file
3. default value